util/treecache/treecache.go

```diff
@@ -16,6 +16,7 @@ package treecache
 import (
 	"bytes"
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
We put stdlib and non-stdlib in separate import groups usually.

On the other hand, in many of the other files, there's no reason to add `github.com/pkg/errors` into its completely separate import group, it can go together with all the other non-stdlib packages.

Reply:
Goimports enforces that. Since it has a Github.com prefix, it is recognized as one needing a separate group.

Reply:
I forgot to goimports that file. Looks like we could add goimports to prometheus' staticcheck job.

Reply:
> Since it has a Github.com prefix, it is recognized as one needing a separate group.

Hmm, but all our other non-stdlib packages are also grouped together just fine. My editor also has goimports, but it doesn't put every GH import into its separate section.

Reply:
So I moved pkg/errors into the other import groups and reran `goimports`. After that, it didn't move it out into a separate group.

`goimports` is strange. I'll make the necessary corrections.

Reply:
Ah right, it does create a new group when adding a completely new import, but when something is already present and in a group it leaves it in that group.